package.json

```diff
@@ -29,23 +29,29 @@
   },
   "homepage": "https://github.com/ca-la/svgthumb-api#readme",
   "dependencies": {
-    "@types/co-body": "0.0.1",
+    "@cala/koa-middleware": "^1.0.3",
     "@types/koa": "2.0.39",
     "@types/koa-router": "7.0.23",
     "@types/node": "8.0.28",
     "@types/node-uuid": "0.0.28",
     "@types/svg2png": "4.1.0",
     "aws-sdk": "2.124.0",
-    "co-body": "5.1.1",
     "koa": "2.3.0",
     "koa-router": "7.2.1",
     "node-uuid": "1.4.8",
     "svg2png": "4.1.1",
     "svgbundler": "1.3.2",
-    "typescript": "2.5.2"
+    "typescript": "^2.9.2"
```

Comment:
@dylanpyle 

I had to bump this to get `@types/sinon` to compile. Should I go through and change the other dependencies to `^` in this PR?

Reply:
@scotttrinh Yeah, that sounds like a plan to me